app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :email, presence: true, format: { with: EMAIL_VALIDATOR,
+                                              message: "invalid. Please enter valid mail id." }
+  validates :email, uniqueness: true
```

Comment:
if user email is not present. User object will have two errors, one for presence and one for uniqueness. Reduce error only to presence one.